test/robust/compare/output1__Stub.js

```diff
@@ -35,12 +63,36 @@ describe('/', function() {
           return;
         }
 
-        body.should.have.property('name');
+        /*eslint-disable*/
+        validator.validate(body, schema).should.be.true;
```

Comment:
JSHint has the `expr` option, there should be one for ESHint too. I guess ESHint works with JSHint options out of the box. Try fixing this by configing the `.eshintrc` file of tests folder.